openapi.yaml

```
@@ -5560,6 +5560,140 @@ paths:
+                  type: boolean
+                  description: 'Enable path prefix stripping in proxy.'
+                  example: true
+              type: object
+      responses:
+        '200':
+          description: 'Application updated successfully.'
+          content:
+            application/json:
+              schema:
+                properties:
+                  uuid: { type: string, description: 'Application UUID.' }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)